{"status": "broken", "uuid": "cd9dcc65-cae0-45fd-b9b1-81c329f7a806", "labels": [{"name": "story", "value": "修改店铺接口"}, {"name": "feature", "value": "店铺管理"}, {"name": "tag", "value": "@pytest.mark.usefixtures('userId')"}, {"name": "suite", "value": "test_set"}, {"name": "subSuite", "value": "TestShop"}, {"name": "parentSuite", "value": "testcase"}, {"name": "host", "value": "DESKTOP-M0PGOTF"}, {"name": "thread", "value": "4156-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "testcase.test_set"}], "stop": 1628559347452, "testCaseId": "fa639c1018f43f439b5e49b145f2052b", "historyId": "d273ddf0d319b10aa905207413d751b2", "start": 1628559347452, "fullName": "testcase.test_set.TestShop#test_shop_update", "statusDetails": {"message": "TypeError: 'NoneType' object has no attribute '__getitem__'", "trace": "@pytest.fixture(scope='session')\n    def userId():\n>       userId = Login().yt_login('易通')['data']['userId']\nE       TypeError: 'NoneType' object has no attribute '__getitem__'\n\ntestcase\\conftest.py:23: TypeError"}, "name": "test_shop_update"}